google/cloud/storage/testbench/gcs_project.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Implement a class to simulate projects (service accounts and HMAC keys)."""
+
```

Comment:
FYI: Some of these scripts only have a single blank line here.

Reply:
For the record (since @devbww already approved). I made all of the scripts in this directory consistent.